ignite/metrics/mean_average_precision.py

```diff
@@ -0,0 +1,420 @@
+from collections import defaultdict
+from typing import Any, Callable, Dict, List, Optional, Set, Tuple, Union
+
+import torch
+
+from ignite.metrics.metric import Metric, reinit__is_reduced, sync_all_reduce
+
+
+def iou(y: torch.Tensor, y_pred: torch.Tensor, crowd: List) -> torch.Tensor:
```

Comment:
Is there a test of this method ? To keep it private (and don't show it in the doc), please use a private naming (like `_iou`)